catalog-rest-service/src/main/resources/json/schema/entity/policies/accessControl/rule.json

```diff
@@ -65,7 +77,7 @@
     "priority": {
       "description": "Priority of this rule among all rules across all policies.",
       "type": "integer",
-      "default": 250000
+      "default": 1000
```

Comment:
We use default as 1000 in data/json and on the UI -> backend contract

@Sachin-chaurasiya after this PR lands, we can remove the priority from UI codebase, so that default is driven by schema.